metpy/plots/declarative.py

```diff
@@ -537,14 +541,63 @@ class MapPanel(Panel):
 
     parent = Instance(PanelContainer)
     layout = Tuple(Int(), Int(), Int(), default_value=(1, 1, 1))
+    layout.__doc__ = """A tuple that contains the description (nrows, ncols, index) of the
+    panel position; default value is (1, 1, 1).
+
+    This trait is set to describe the panel position and the default is for a single panel. For
+    example, a four-panel plot will have two rows and two columns with the tuple setting for
+    the upper-left panel as (2, 2, 1), upper-right as (2, 2, 2), lower-left as (2, 2, 3), and
+    lower-right as (2, 2, 4). For more details see the documentation for Matplotlib subplots.
+    """
+
     plots = List(Any())
+    plots.__doc__ = """A list of handles that represent the plots (e.g., ContourPlot,
+    FilledContourPlot, ImagePlot) to put on a given panel.
```

Comment:
If you use `` `ContourPlot` ``, etc. links to those classes will be generated.

Reply:
Neat. Done.